draft-ietf-jsonpath-base.md

```diff
@@ -1309,96 +1230,99 @@ Queries:
-| `$[0, 3]` | `"a"` <br> `"d"` | `$[0]` <br> `$[3]` | Indices |
-| `$[0:2, 5]` | `"a"` <br> `"b"` <br> `"f"` | `$[0]` <br> `$[1]` <br> `$[5]` | Slice and index |
-| `$[0, 0]` | `"a"` <br> `"a"` | `$[0]` <br> `$[0]` | Duplicated entries |
-| `$[7]["n", "p"]` | `"v"` <br> `"x"` | `$[7]['n']` <br> `$[7]['p']` | Dot child |
-| `$[? @ <= "b" || @ >= "g", 2]` | `"a"` <br> `"b"` <br> `"g"` <br> `"c"` | `$[0]` <br> `$[1]` <br> `$[6]` <br> `$[2]` | Filter and index |
-{: title="List selector examples"}
+Selectors of different kinds may be combined within a single child __APPENDER__.
+
+The resulting nodelist of a child __APPENDER__ is the concatenation of
+the nodelists from each of its selectors in the order that the selectors
+appear in the list.
+Note that any node matched by more than one selector is kept
+as many times in the nodelist.
 
-### Descendant Selectors
+### Descendant __APPENDER__
 
 #### Syntax
 {: unnumbered}
 
-The descendant selectors start with a double dot `..` and can be
-followed by an object member name (similar to the `dot-selector`),
-a wildcard (similar to the `dot-wild-selector`),
-an `index-selector`, `index-wild-selector`, `filter-selector`, or `list-selector` acting on objects or arrays,
-or a `slice-selector` acting on arrays.
+The descendant __APPENDER__ consists of a double dot `..`
+followed by a child __APPENDER__ (`descendant-APPENDER`).
+
+Shortand notations are also provided that correspond to the shorthand forms of the child __APPENDER__.
 
 ~~~~ abnf
-descendant-selector = ".." ( dot-member-name      /  ; ..<name>
-                             wildcard             /  ; ..*
-                             index-selector       /  ; ..[<index>]
-                             index-wild-selector  /  ; ..[*]
-                             slice-selector       /  ; ..[<slice-index>]
-                             filter-selector      /  ; ..[<filter>]
-                             list-selector           ; ..[<list-entry>,...]
-                           )
+descendant-APPENDER              = (descendant-child /
+                                    descendant-wildcard-shorthand /
+                                    descendant-member-name-shorthand)
+descendant-child                 = ".." child-APPENDER
+
+descendant-wildcard-shorthand    = ".." wildcard
+descendant-member-name-shorthand = ".." dot-member-name
 ~~~~
 
-Note that `..` on its own is not a valid selector.
+The `descendant-wildcard-shorthand` is shorthand for `..[*]`.
+
+A `descendant-member-name-shorthand` of the form `..<member-name>` is shorthand for `..['<member-name>']`.
+
+Note that `..` on its own is not a valid __APPENDER__.
 
 #### Semantics
 {: unnumbered}
 
-A descendant selector selects zero or more descendants of a node.
+<!-- The following does not address https://github.com/ietf-wg-jsonpath/draft-ietf-jsonpath-base/issues/252 -->
+
+A descendant __APPENDER__ selects zero or more descendants of the input value.
```

Comment:
I don't think a segment "selects".  It applies its selector to each of the self-or-descendant axis nodes (we need more words).  The sequence of application is partially deterministic.  Outputs of the selectors are concatenated in the sequence of application.

Reply:
I avoided saying "selects" in d5286be84535a41c0aa8e789b1c2d58c5cec5d86.